### PR TITLE
feat(GradientOpacitySlider): Rewrite GradientOpacitySlider and update style

### PR DESCRIPTION
**Pull request by labdala**
-  Re-write GradientOpacitySlider using bootstrap;
 - Change `style.css` selector's properties to have wider sliders, and to enable vertical slider

To-do: optimize GradientOpacitySlider to avoid unnecessary re-renders

### CODE REVIEW

File: src/Images/GradientOpacitySlider.jsx [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger'
+import Tooltip from 'react-bootstrap/Tooltip'
+import cn from 'classnames'
 import '../style.css'
+import applyContrastSensitiveStyleToElement from '../applyContrastSensitiveStyleToElement'
 
-function GradientOpacitySlider(props) {
+const GradientOpacitySlider = React.memo(function GradientOpacitySlider(props) {
   const { service } = props
+  const state = useSelector(service, (state) => state)
```
Comment (PaulHax) [inline src/Images/GradientOpacitySlider.jsx:15]: Select state component needs.

**Review by PaulHax — APPROVED** [review]

**Comment by labdala** [comment]
This pull request has already been approved. @PaulHax, could you please confirm whether you think this branch is ready to be merged since I made some changes?

**Review by PaulHax — CHANGES_REQUESTED** [review]
When I drag the sliders, the ticks don't move.  Hmmm.  Something is broken in the state update flow.  Check that the component re-renders and the state value changes.

The `IMAGE_GRADIENT_OPACITY_CHANGED` slider does not seem to update the rendering.

File: src/Images/GradientOpacitySlider.jsx [hunk 2]
```diff
@@ -40,63 +46,70 @@ function GradientOpacitySlider(props) {
...
-          min={0}
-          max={1}
-          value={actorContext.gradientOpacity}
-          step={0.01}
-          onChange={(_e, val) => {
-            opacitySliderChanged(val)
-          }}
-        />
+        <Form>
+          <Form.Group>
```
[the remark this answers is not in the document]
Comment (labdala) [reply src/Images/GradientOpacitySlider.jsx:76]: Yes and right. The first problem is related to the use of React.memo again. I just deleted it.

**Review by PaulHax — APPROVED** [review]
Lets do it!

**Comment by labdala** [comment]
Thank you so much for your suggestions @PaulHax !

**Pull request merged**